### ISSUE TITLE
imagePullSecrets property in EventBus.spec.nats.native not being passed to StatefulSet

### ISSUE THREAD

**Issue by zephinzer** [argoproj/argo-events#969]
**Describe the bug**
The `spec.nats.native.imagePullSecrets` should pass the Secret reference to the StatefulSet, but it does not

**To Reproduce**
1. Create an EventBus resource
```yaml
apiVersion: argoproj.io/v1alpha1
kind: EventBus
metadata:
  name: default
spec:
  nats:
    native:
      imagePullSecrets:
      - name: dockerhub-credentials
      # Optional, defaults to 3. If it is < 3, set it to 3, that is the minimal requirement.
      replicas: 3
      # Optional, authen strategy, "none" or "token", defaults to "none"
      auth: token
```
2. Apply above EventBus resource
3. Observe that the created StatefulSet does not have the `imagePullSecrets` property defined in the spec

**Expected behavior**
The `imagePullSecrets` should have been passed down

**Screenshots**
N/A

**Environment (please complete the following information):**
 - Kubernetes: v1.10.5
 - Kubectl: v1.19.0
 - Argo Events: v1.0.0 (installed via Helm)

**Additional context**
N/A

---

**Message from the maintainers**:

If you wish to see this enhancement implemented please add a 👍 reaction to this issue! We often sort issues this way to know what to prioritize.

**Comment by zephinzer** [comment]
Manually editing the StatefulSet and inserting the `imagePullSecrets` property works- manually editing the EventBus resource does not, so I believe it's an issue with the operator not passing it to the generated StatefulSet resource?

**Comment by whynowy** [comment]
This is a new feature just in `master`, it will be released in v1.2.

**Comment by whynowy** [comment]
https://github.com/argoproj/argo-events/pull/947
